verify.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+flake8 couchbase.py test_couchbase.py metric_info.py
```

Comment:
add set -ex so that you see what is happening and exit if there are errors.  currently flake8 is failing but verify.sh is exiting with 0 and travis things it is succeeding.